### DISCUSSION TITLE
休斯敦 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#421] [General]
# 休斯敦 - Miao Yu | 于淼

https://yufree.cn/cn/2023/06/10/houston/

**Comment by earfanfan** [comment]
哇，这篇博客读下来感觉像是跟着三水出去玩了一样。万万没想到三水的拍照水平还挺好的，不比我差。那些宝石 blingbling 的好好看，它们有名字吗？

**Reply by yufree** [reply]
有，但我没记，反正也都是新词不认识，除了博物馆我也不大可能再见这些石头，一期一会。

**Reply by earfanfan** [reply]
我问你那些石头的名字，本来是以为石头们很好看，看你写的矿石宝石展，又以为石头们也不贵，想买了送给朋友。既然你不记得，那我就自己动手在有限的范围内搜索了一下，你看的是不是这个<https://www.hmns.org/exhibits/permanent-exhibitions/> Cullen Hall of Gems and Minerals？又搜索了一通，找到一个名字，萤石（Fluorite）。哈哈，我上淘宝搜了搜 萤石 石头，发现真得很便宜，就是没有你拍的那些石头那么好看。

话又说回来矿石和宝石的区别是撒？我的理解是宝石属于矿石，只不过宝石是矿石中被人们挑出来的，符合人们审美的更好看的石头，而且因为稀少更加宝贵，所以称为宝石……

**Reply by earfanfan** [reply]
我忽然明白，让我觉得好看的不是石头本身，是石头折射出来的光。那买个三棱镜也能看到好看的光了，就是不像你拍的石头那样是发散状的。

**Reply by yufree** [reply]
这是展台灯光布置的效果，跟宝石加工切削也有关系，萤石只是一种，类似天青石、紫水晶、蛋白石这些天然矿石都很漂亮也不贵，这个爱好可是个深坑。
